b1-base.xml

```diff
@@ -439,18 +452,70 @@ B-1B set file.
                         </binding>
                     </item>
                 </menu>
+                <menu n="101">
+					<label>FDM Tuning</label>
+					<item>
+						<label>Tune Drag</label>
+						<binding>
+							<command>nasal</command>
+							<script>dialog.tuner_drag_dialog.open()</script>
+						</binding>
+					</item>
+					<item>
+						<label>Tune Lift</label>
+						<binding>
+							<command>nasal</command>
+							<script>dialog.tuner_lift_dialog.open()</script>
+						</binding>
+					</item>
+					<item>
+						<label>Tune Pitch</label>
+						<binding>
+							<command>nasal</command>
+							<script>dialog.tuner_pitch_dialog.open()</script>
+						</binding>
+					</item>
+					<item>
+						<label>Tune Roll</label>
+						<binding>
+							<command>nasal</command>
+							<script>dialog.tuner_roll_dialog.open()</script>
+						</binding>
+					</item>
+					<item>
+						<label>Tune Side</label>
+						<binding>
+							<command>nasal</command>
+							<script>dialog.tuner_side_dialog.open()</script>
+						</binding>
+					</item>
+					<item>
+						<label>Tune Yaw</label>
+						<binding>
+							<command>nasal</command>
+							<script>dialog.tuner_yaw_dialog.open()</script>
+						</binding>
+					</item>
+				</menu>
             </default>
         </menubar>
 
         <systems>
-            <electrical>
-                <name>electrical</name>
-                <number>0</number>
-                <path>Aircraft/B-1B/Systems/generic-electrical.xml</path>
-            </electrical>
-            <autopilot>
+            <autopilot n="0">
                 <path>Aircraft/B-1B/Systems/b1b-autopilot.xml</path>
             </autopilot>
+            <autopilot n="1">
+                <path>Aircraft/B-1B/Systems/engines.xml</path>
+            </autopilot>
+            <property-rule n="110">
+                <path>Aircraft/B-1B/Systems/filters.xml</path>
+            </property-rule>
+            <property-rule n="200">
+                <path>Systems/annunciators.xml</path>
+            </property-rule>
+            <property-rule n="200">
```

Comment:
does this need to be a different number than the annunciators?

Reply:
I'm not actually to sure what that number represents. However I do know that the file is being called upon and is working, so the fact that it's the same number as computers.xml isn't an issue I don't think? When you say "Different number than the annunciators" Do you mean the amount of annunciators?

Reply:
Basically, each definition should be assigned a number, i.e. just above this line you have autopilot n=0 and n=1. These numbers should be unique for each definition.